math/check_amicable_pair.cpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 @author iamnambiar
```

Comment:
```suggestion
```

Reply:
I took the style from previously accepted code. Anyway I'll clear that.